commit 2dc145ec2c214c46ff2885a094ac24d9b9407c94
Author: Montel Laurent <montel@kde.org>
Date:   Sun Mar 22 18:23:19 2015 +0100

    Search kiocore

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b02f803..0d286ca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -31,6 +31,7 @@ find_package(KF5 REQUIRED COMPONENTS
     NewStuff
     Notifications
     XmlGui
+    KIO
 )
 
 find_package(LibKEduVocDocument REQUIRED)
